src/App.css

```diff
@@ -1,3 +1,6 @@
 .App {
   text-align: center;
+  display: grid;
+  height: 100vh;
+  grid-template-rows: 50px 1fr 50px;
```

Comment:
Try using em/rem relative units instead of pixel units
https://css-tricks.com/rem-global-em-local/

Reply:
Thanks ! I am still getting used to using rems instead of pixels ! Muscle memory with the px ..